config.js

```diff
@@ -1,11 +1,7 @@
-const AWS = require('aws-sdk');
 require('dotenv').config();
 
-const config = new AWS.Config({
-  // hardcode the config for now, DO NOT PUSH
```

Comment:
Remove this comment, since we already fixed that

Reply:
On my end it is removed? I'll try re-committing the file.